packages/databases-collections/src/components/rename-collection-modal/rename-collection-modal.tsx

```diff
@@ -109,7 +109,7 @@ function RenameCollectionModal({
   useTrackOnChange(
     (track: TrackFunction) => {
       if (isVisible) {
-        track('Screen', { name: 'rename_collection_modal' });
+        track('Screen', { name: 'rename_collection_modal' }, undefined);
```

Comment:
ditto

Reply:
Thoughts on this? I still think the best would be to keep things as they are here, and have dedicated events instead of `Screen`